canvas_course_admin_tools/settings/local.py

```diff
@@ -29,6 +29,12 @@
 dictConfig(LOGGING)
 
 SELENIUM_CONFIG = {
+    'class_roster': {
+        'course_link': 'https://canvas.dev.tlt.harvard.edu/courses/3787'
```

Comment:
maybe this should be built from the canvas_base_url in the code, like the other links (see manage_course's `relative_url`, below)? That way any change to the base URL in the future (including running selenium tests on qa, stage, etc) will be picked up automatically.